packages/calcite-components/src/utils/date.ts

```diff
@@ -151,7 +151,10 @@ export function datePartsFromLocalizedString(
  */
 export function dateToISO(date?: Date): string {
   if (date instanceof Date) {
-    return new Date(date.getTime() - date.getTimezoneOffset() * 60000).toISOString().split("T")[0];
```

Comment:
Can you also update the spec test to add coverage for the failing scenario?

Reply:
No longer applicable based on: https://github.com/Esri/calcite-design-system/pull/9424/files#r1622784980.